File: Python/Usages of Arithmetic operator/Multiple usages of arithmetic operators in one program.py
print("('+'=addition), ('-'=addition), ('x'or'*'=multiplication), ('/'=division), ('%'=modulus)")
operation = input("Select the operation: ")[0]

num1 = int(input("Enter the first number: "))
num2 = int(input("Enter the second number: "))
sum = 0; flag = True

if operation == '+':
    sum = num1 + num2;
elif operation == '-':
    sum = num1 - num2
elif operation == 'X' or operation == 'x' or operation == '*':
    sum = num1 * num2
elif operation == '/':
    sum = num1 / num2
elif operation == '%':
    sum = num1 % num2
else:
    flag = False

if flag == True:
    print("The result is = ", sum)
else:
    print("Error! Invalid operation")

File: Java/Usages of Arithmetic operator/Multiple usages of arithmetic operators in one program.java
import java.util.Scanner;
class arithmeticOperators {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("('+'=addition), ('-'=addition), ('x'or'*'=multiplication), ('/'=division), ('%'=modulus)");
        System.out.print("Select the operation: ");
        char operation = input.next().charAt(0);

        System.out.print("Enter the first number: ");
        int num1 = input.nextInt();
        System.out.print("Enter the second number: ");
        int num2 = input.nextInt();
        int sum = 0; boolean flag = true;

        if (operation == '+') {
            sum = num1 + num2;
        } else if (operation == '-') {
            sum = num1 - num2;
        } else if (operation == 'X' || operation == 'x' || operation == '*') {
            sum = num1 * num2;
        } else if (operation == '/') {
            sum = num1 / num2;
        } else if (operation == '%') {
            sum = num1 % num2;
        } else {
            flag = false;
        }

        if (flag == true) {
            System.out.println("The result is = " + sum);
        } else {
            System.out.println("Error! Invalid operation");
        }
    }
}